lib/config/presets/internal/replacements.ts

```diff
@@ -51,6 +52,18 @@ export const presets: Record<string, Preset> = {
     ],
     ignoreDeps: [], // Hack to improve onboarding PR description
   },
+  'airbnb-prop-types-to-prop-types-tools': { // eslint-disable-line sort-keys
```

Comment:
```suggestion
  'airbnb-prop-types-to-prop-types-tools': {
    // eslint-disable-line sort-keys
```

I think this should fix the Prettier lint error? I hope that preserves the ESLint ignore though...

Reply:
it won’t preserve the ignore. I think you need disable-next-line, on the previous line.

Reply:
```suggestion
// eslint-disable-next-line sort-keys
  'airbnb-prop-types-to-prop-types-tools': { // eslint-disable-line sort-keys
```

Let's try this then... :upside_down_face: